docs: [DEVX-6059] Make it clearer how to use secrets on the website

namespacebrian opened the pull request:
The README didn't even tell the customer how to use secrets in their site code.  It made a passing reference to `pantheon_get_secret()`, and to "the SDK" without any link or further information about _what SDK_.

This PR improves that.

namespacebrian committed:
Make it clearer how to use secrets on the website

greg-1-anderson commented:
Tests are failing because Composer is detecting security vulnerabilities in the fixtures. It might be best to disable this feature in tests.

namespacebrian committed:
Early access language adjustments, don't commit .DS_Store

- Remove language from README implying active development (this is false)
- Remove language implying customers must request access

greg-1-anderson commented on README.md:
```diff
@@ -554,6 +557,69 @@ EOF
 
+## Using Secrets in Your Site Code
+
+Once you've set secrets with the `runtime` type and `web` scope, you can retrieve them in your site's PHP code.
```
Secrets of any type can be fetched via the API if they have 'web' scope.

namespacebrian replied:
removed.

namespacebrian committed:
composer update

namespacebrian committed:
Clarify that runtime scope is not required

greg-1-anderson commented on README.md:
```diff
@@ -554,6 +557,69 @@ EOF
 `terminus secret:site:set ${SITE_NAME} COMPOSER_AUTH ${COMPOSER_AUTH_JSON} --type=env --scope=user,ic`
```
Both ` and ``` used. Not important. I'm just sort of wondering why Github is rendering all of the text above with a red background. Not sure why. Not important.

namespacebrian replied:
I removed the redundant backticks.

idk why it's showing red, but from the diff I'm seeing, it looks like it was already highlighting red before these changes... no idea why.

<img width="2566" height="757" alt="image" src="https://github.com/user-attachments/assets/27876ba9-c5c7-44f7-9880-b3dbc96f74e4" />

namespacebrian committed:
Remove redundant backticks

namespacebrian committed:
Revert "composer update"

This reverts commit 9a777ff993de41d912c0115e7250f1440d20e9e2.
